container_test.go

```diff
@@ -12,9 +12,12 @@ import (
 	"testing"
 	"time"
 
+	"github.com/docker/go-connections/nat"
 	"github.com/stretchr/testify/assert"
 
 	"github.com/testcontainers/testcontainers-go/wait"
+
+	_ "github.com/lib/pq"
```

Comment:
Do we need the import (and the dependency)?

I'll double check it fetching the PR locally